src/scf/scf_solvers.jl

```diff
@@ -57,7 +60,7 @@ CROP-accelerated root-finding iteration for `f`, starting from `x0` and keeping
 a history of `m` steps. Optionally `warming` specifies the number of non-accelerated
 steps to perform for warming up the history.
 """
-function CROP(f, x0, m::Int, max_iter::Int, tol::Real, warming=0)
+function CROP(f, x0, info0, m::Int, max_iter::Int, tol::Real, warming=0)
```

Comment:
It's useful to have since it parallels the anderson implementation, and we might want to look at it again in 5 years. Why can't `info` be an optional arg (or kwarg) so that solvers don't have to bother with it?

Reply:
If we want the solvers to be able to use the isconverged callback it somehow needs an info. But perhaps we can have a default-scfres function, which would be the default for the arg.